lib/src/parsers/_national_number_parser.dart

```diff
@@ -35,6 +40,14 @@ abstract class NationalNumberParser {
     final nationalPrefixForParsing = patterns.nationalPrefixForParsing;
     final transformRule = patterns.nationalPrefixTransformRule;
 
+    final transformPrefix = patterns.nationalPrefixTransformRule
+        ?.replaceAllMapped(RegExp(r'(\$\d)'), (_) => '');

```

Comment:
could you add an example of what this would do for your personal case as a comment above this line ? Thanks

Reply:
Added more comments ✅  (in slightly different place however, I could manage to fit it in here - the meaning did not match with the code)